### PR TITLE
Fix battery level in shipment section

### PR DESCRIPTION
**Pull request by TanmayJain92**
## Purpose
Fix tracker battery level in shipment page.

### CODE REVIEW

File: src/utils/constants.js [hunk 1]
```diff
@@ -1887,7 +1887,16 @@ export const getShipmentFormattedRow = (
     }
 
     if (!_.isEmpty(sensorReports)) {
-      const reports = _.take(_.orderBy(_.filter(sensorReports, { shipment_id: editedShipment.partner_shipment_id }), 'create_date', 'desc'), 10);
+      const reports = _.take(
+        _.orderBy(
+          _.filter(sensorReports, { shipment_id: editedShipment.partner_shipment_id }),
+          [
+            (obj) => moment(obj.report_entry.report_timestamp),
```
Comment (patelradhika) [inline src/utils/constants.js:1894]: Sort by activation_date

**Review by patelradhika — CHANGES_REQUESTED** [review]

**Comment by patelradhika** [comment]
@TanmayJain92 Also add bump patch version in package.json file

File: src/utils/constants.js (hunk 1 above)
Comment (TanmayJain92) [reply src/utils/constants.js]: done

**Comment by TanmayJain92** [comment]
> @TanmayJain92 Also add bump patch version in package.json file
Done.

**Review by patelradhika — APPROVED** [review]
Looks good

**Pull request merged**